vbox-2012r2-wmf5.json

```diff
@@ -3,7 +3,6 @@
     {
       "type": "virtualbox-iso",
       "vboxmanage": [
-        [ "modifyvm", "{{.Name}}", "--natpf1", "guest_winrm,tcp,,55985,,5985" ],
```

Comment:
I'm not sure these `modifyvm`s should go. They have not been tweaked along with the below winrm setting.